proto/iotics/api/interest.proto

```diff
@@ -22,21 +23,66 @@ option php_namespace = "Iotics\\Api";
 
 // InterestAPI enables creation and management of interests between a twin and a feed.
 service InterestAPI {
-  // Fetch feed data for this interest.
+  // Fetch feed data for this interest. (local and remote)
   rpc FetchInterests(FetchInterestRequest) returns (stream FetchInterestResponse) {}
 
-  // Fetch last data shared on this interest.
+  // Fetch last data shared on this interest. (local and remote)
   rpc FetchLastStored(FetchLastStoredRequest) returns (FetchInterestResponse) {}
 
   // List all interests associated to a given follower twin (Not implemented yet).
   rpc ListAllInterests(ListAllInterestsRequest) returns (ListAllInterestsResponse) {}
 
-  // Create an interest between a follower twin and a followed feed.
+  // Create an interest between a follower twin and a followed feed. (Not implemented yet).
   rpc CreateInterest(CreateInterestRequest) returns (CreateInterestResponse) {}
 
-  // Delete an existing interest.
+  // Delete an existing interest. (Not implemented yet).
   rpc DeleteInterest(DeleteInterestRequest) returns (DeleteInterestResponse) {}
 
+  // Send a message to an input. (local and remote)
+  rpc SendInputMessage(SendInputMessageRequest) returns (SendInputMessageResponse) {}
+
+}
+
+// An interest is the relationship between a Twin and an Input.
+message InputInterest {
+  // InputInterest destination input identification.
+  message DestinationInput {
+    // Input to send the message to
+    Input input = 1;
+    // HostID to identify a remote input (Optional, keep empty if input is local)
+    HostID hostId = 2;
+  }
+  // Sender twin unique identifier.
+  TwinID senderTwinId = 1;
+
+  // a reference to the interested feed
```

Comment:
a reference to the input of interest

Reply:
https://github.com/Iotic-Labs/api/pull/16/commits/df0ad530ea4aa05433fb5b5d64770ea93a6b0224